postgres/datadog_checks/postgres/config.py

```diff
@@ -44,6 +44,7 @@ def __init__(self, instance):
             raise ConfigurationError("Application name can include only ASCII characters: %s", self.application_name)
 
         self.query_timeout = int(instance.get('query_timeout', 5000))
+        self.idle_connection_timeout = int(instance.get('idle_connection_timeout'), 60000)
```

Comment:
This default may need discussion. I chose 60s to really only close connections to databases very infrequently accessed/sampled. But we might want to more aggressively close some of these?